src/ensembl/src/content/app/entity-viewer/EntityViewer.tsx

```diff
@@ -66,8 +66,14 @@ export type EntityViewerParams = {
 };
 
 const client = new ApolloClient({
-  uri: '/toygraphql',
-  cache: new InMemoryCache()
+  uri: 'http://hx-rke-wp-webadmin-13-worker-1.caas.ebi.ac.uk:31497',
```

Comment:
**NOTE TO SELF**: remember to change to `/thoas` before merging. Database on the web instance of Thoas needs to be updated (after Kieron has fixed the 16MB document).